server/api/level.js

```diff
@@ -33,7 +33,9 @@ function Level(email, password, cb, mock) {
       this.user.uuid = 1110568334;
       this.user.token = '1CA902A8E5D74EDBB0701B8DA5A79DB6';
     }
-    cb();
```

Comment:
This causes mocks to call their callbacks before the instance is instantiated, and results in not having access to prototypal methods. It needs to be delayed (`setTimeout`).